services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -1,64 +1,69 @@
 package cromwell.pipeline.service
 
-import cromwell.pipeline.datastorage.dao.repository.{ Dao, ProjectRepository }
+import cromwell.pipeline.datastorage.dao.repository.{Dao, ProjectRepository}
 import cromwell.pipeline.datastorage.dao.repository.utils.TestProjectUtils
-import cromwell.pipeline.datastorage.dto.{ Project, ProjectAdditionRequest }
-import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, Matchers }
+import cromwell.pipeline.datastorage.dto.{Project, ProjectAdditionRequest, ProjectId, UserId}
+import org.scalatest.{AsyncWordSpec, BeforeAndAfterAll, Matchers}
 import org.scalatestplus.mockito.MockitoSugar
 import org.mockito.Matchers.any
 import org.mockito.Mockito.when
 
 import scala.concurrent.Future
 
-class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar with BeforeAndAfterAll {
+class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
 
   private val projectRepository = mock[ProjectRepository]
   private val projectService = new ProjectService(projectRepository)
-  private val dummyProject: Project = TestProjectUtils.getDummyProject()
 
   "ProjectServiceTest" when {
 
     "addProject" should {
-      "return id of a new project" taggedAs (Service) in {
+      "return id of a new project" in {
```

Comment:
Why did you remove tagging?

Reply:
It was misss-click. I will return it back.

Reply:
done